### PR TITLE
Beta2 release

### PR DESCRIPTION
**Pull request by AsterAI**
Reduced phalcon syntax errors from 30 to 16 
Fixed completion bug when invoking AC in method definition block

[Zephir.zip](https://github.com/phalcon/zephir-idea-plugin/files/1036677/Zephir.zip)

### CODE REVIEW

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     ]]></description>
 
   <change-notes><![CDATA[
-        <li><b>0.3.2-beta1</b>: Introduced class members in completion list</li>
+        <li><b>0.3.2-alpha</b>: Introduced class members in completion list, improved syntax support</li>
```
Comment (sergeyklay) [inline resources/META-INF/plugin.xml]: Actually there is `0.3.2-beta1`: https://github.com/phalcon/zephir-idea-plugin/releases/tag/0.3.2-beta1

I propose to release next beta version, `0.3.2-beta2`

**Comment by sergeyklay** [comment]
@AsterAI Could you take a look

File: resources/META-INF/plugin.xml (hunk 1 above)
Comment (AsterAI) [inline resources/META-INF/plugin.xml]: Ok, wait a minute please

**Comment by AsterAI** [comment]
@sergeyklay  done

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**